dotnet/src/libraries/AuthorizedStore.Fake/Stores/StoreService.cs

```diff
@@ -68,6 +73,9 @@ public async Task<Store> UpdateAsync(int id, Store entity)
             StringExtensions.CheckNullOrWhiteSpace(entity?.ContractContent, nameof(Store.ContractContent));
             CheckInvalidName(entity?.Name);
             await CheckDuplicate(entity?.Name, id);
+            entity.Category = entity.Category?.Id != null
```

Comment:
```suggestion
            entity.Category = entity.Category != null
```
同前所述，判斷至 `entity.Category` 即可， `int` 不會為 null。